### ISSUE TITLE
High CPU usage on Mac Builds running on El Capitan

### ISSUE THREAD

**Issue by foopod** [inkle/inky#63]
<img width="285" alt="screen shot 2016-09-20 at 9 39 38 am" src="https://cloud.githubusercontent.com/assets/3768616/18650071/1c4e770e-7f17-11e6-9dc5-e62822839f98.png">
The Inky Editor runs very slowly due to high CPU usage. I have tried both the 0.4.0 and 0.4.1 builds, but the issue persists. This only happens when working from a saved file, if I open the editor with a new file everything runs fine until I save it.

OSX El Capitan (10.11.6)
MacBook Pro (Retina, 13-inch, Early 2015)
2.7 GHz Intel Core i5
8 GB 1867 MHz DDR3

**Comment by foopod** [comment]
Cloned the Repo and build on my system. Seems to run fine.

Don't know much about the project so bare with me, but Electron & Electron Helper seem to have much lower cpu usage now. The UI is totally responsive unlike before.
<img width="283" alt="screen shot 2016-09-20 at 12 52 54 pm" src="https://cloud.githubusercontent.com/assets/3768616/18653850/eb56f4f4-7f30-11e6-8959-4d818c74cb68.png">

Will try packaging it up on my machine and see if that continues to work, or maybe that is where the problem is.

**Comment by foopod** [comment]
The packaged version won't let me do anything and keeps spewing the below error.
<img width="418" alt="screen shot 2016-09-20 at 1 00 53 pm" src="https://cloud.githubusercontent.com/assets/3768616/18654037/304beb54-7f32-11e6-99d6-635baf8d7c6a.png">

I am actually having more fun doing this that I was writing my story hahaha

**Comment by joethephish** [comment]
Yeah, I reverted a change in one of the syntax highlighter regexes which seemed to be causing major performance problems, just need to get that into a new release build.

Not sure about the packaged build breaking! I'll give it a try.

**Comment by joethephish** [comment]
Odd - I'm not having this problem when packaging latest code, even after re-cloning the repo and re-building/packaging from scratch.

**Comment by davekki** [comment]
I just wanted to add some extra information that may or may not help with this - we have 2 iMacs and 2 Macbook Pros here and inky works fine on both iMacs, and has the 170%+ CPU usage issues on the Macbook Pros. One of the Macbook Pros is a 15" with discreet Radeon GPU, the 13" one is just Intel built-in. As the OP mentions it only runs at 170%+ CPU usage when you try and edit a saved file. Cutting and pasting into a new ink file has no issues. I've downloaded and rebuilt on the Macbook Pro 15" and it still has that CPU problem with saved files. Any ideas what could be causing it? 
Cheers, Tim.
